frontend/public/components/storage/attach-storage.tsx

```diff
@@ -0,0 +1,499 @@
+    return (
+      <div className="co-m-pane__body">
+        <Helmet>
+          <title>{title}</title>
+        </Helmet>
+        <form
+          className="co-m-pane__body-group co-create-secret-form"
+          onSubmit={this.save}
+        >
+          <h1 className="co-m-pane__heading">{title}</h1>
+          <p className="co-m-pane__explanation">
+            {this.props.explanation}
+            &nbsp;
+            <ResourceIcon kind={getKindFromPlural(this.props.kind)} />
+            <Link
+              to={formatNamespacedRouteForResource(
+                `${this.props.kind}/${this.props.name}`
+              )}
+              id="linkToResource"
+            >
+              {this.props.name}.
+            </Link>
+          </p>
+
+          <fieldset disabled={!this.props.isCreate}>
+            <div className="form-group">
+              <label
+                className="control-label co-required"
+                htmlFor="volume-name"
+              >
+                Storage
+              </label>
+              <PvcDropdown
+                namespace={this.props.namespace}
+                onChange={this.handlePvcChange}
+                id="claimName"
+                name="claimName"
+                selectedKey={this.state.formData.claimName}
+                required
+              />
+            </div>
+            <div className="form-group">
+              <label className="control-label" htmlFor="volume-name">
+                Volume Name
+              </label>
+              <div>
+                <input
+                  className="form-control"
+                  type="text"
+                  onChange={this.handleChange}
+                  aria-describedby="volume-name-help"
+                  id="volume-name"
+                  name="volumeName"
+                  value={this.state.formData.volumeName}
+                  pattern="[a-z0-9](?:[-a-z0-9]*[a-z0-9])?"
+                />
+                <p className="help-block" id="volume-name-help">
+                  Unique name to identfiy this volume. If not specified, a
+                  volume name is generated.
+                </p>
+              </div>
+            </div>
+            <div className="form-group">
+              <label className="control-label" htmlFor="mount-path">
+                Mount Path
+              </label>
+              <div>
+                <input
+                  className="form-control"
+                  type="text"
+                  onChange={this.handleChange}
+                  aria-describedby="mount-path-help"
+                  name="mountPath"
+                  id="mount-path"
+                  value={this.state.formData.mountPath}
+                />
+                <p className="help-block" id="mount-path-help">
+                  Mount path for the volume inside the container. If not
+                  specified, the volume will not be mounted automatically.
```

Comment:
> If not specified, the volume will not be mounted automatically.

I know this is how things worked in the previous console, but I'm not sure of the value of this, particularly since you can only fix by editing YAML later. I'm wondering if we should just make this a required field.

@erinboyd Opinion?

Reply:
@spadgett @zherman0 I think it makes sense to require it. It's worthless without it. Better to pave a path for success than just configure it and let it fail imo